Readme.markdown

```diff
@@ -9,7 +9,7 @@ Simple static table views for iOS in Swift. Static's goal is to separate model d
 
 | Swift Version | Static Version |
 | ------------- | -------------- |
-| 4.2+          | 3.0.1          |
```

Comment:
This line should be kept because v3.0.3 is the last version available with Swift 4.2 🙂